dev/benchmarks/macrobenchmarks/lib/src/filtered_child_animation.dart

```diff
@@ -27,25 +27,21 @@ class FilteredChildAnimationPage extends StatefulWidget {
 }
 
 class _FilteredChildAnimationPageState extends State<FilteredChildAnimationPage> with SingleTickerProviderStateMixin {
-  AnimationController _controller;
+  late final AnimationController _controller = AnimationController(vsync: this, duration: const Duration(seconds: 2));
   final GlobalKey _childKey = GlobalKey(debugLabel: 'child to animate');
   Offset _childCenter = Offset.zero;
 
-  FilterType _filterType;
-  bool _complexChild;
-  bool _useRepaintBoundary;
+  late FilterType? _filterType = widget.initialFilterType;
+  late bool _complexChild = widget.initialComplexChild;
+  late bool _useRepaintBoundary = widget.initialUseRepaintBoundary;
```

Comment:
Maybe initialize these to false, but keep the widget initialization in `initState`